src/services/MailService.js

```diff
@@ -12,7 +12,7 @@ const develop = process.env.NODE_ENV === 'development'
 
 const projectRoot = getProjectRoot()
 
-type RendererType = (Object) => string
+type RendererType = (mixed) => string
```

Comment:
mixed should not be used. I suppose this is because our forms do not have types.
You can create a task for this though. Not sure if its possible to add types for forms.